definitions/checks/foreman/check_checkpoint_segments.rb

```diff
@@ -0,0 +1,50 @@
+module Checks
+  module Foreman
+    class CheckpointSegments < ForemanMaintain::Check
+      metadata do
+        label :check_checkpoint_segments
+        description 'Check if checkpoint_segments parameter persent in custom hiera'
+        tags :pre_upgrade
+      end
+
+      HIERA_FILE = '/etc/foreman-installer/custom-hiera.yaml'.freeze
+
+      def run
+        files = []
+        files << feature(:foreman_database).config_files[0] if check_postgres_config
```

Comment:
It would be better if methods (i.e  check_postgres_config & check_custom_hiera) itself returns file paths when config present otherwise `nil`?